library/core/src/slice/mod.rs

```diff
@@ -5390,7 +5390,7 @@ unsafe impl<'a, T> TrustedRandomAccess for ChunksMut<'a, T> {
 /// This struct is created by the [`chunks_exact`] method on [slices].
 ///
 /// [`chunks_exact`]: ../../std/primitive.slice.html#method.chunks_exact
```

Comment:
This is #63351 , no need to change anything.

Reply:
this applies to all primitives right? like this:
```
/// [slices]: ../../std/primitive.slice.html
```

Reply:
Not all primitives, but to all methods on slices. See the issue for a full list; it's mostly slices and pointers.